hack/jenkins/e2e.sh

```diff
@@ -534,6 +534,7 @@ case ${JOB_NAME} in
           ${GCE_FLAKY_TESTS[@]:+${GCE_FLAKY_TESTS[@]}} \
           ${GCE_SLOW_TESTS[@]:+${GCE_SLOW_TESTS[@]}} \
           ${TRUSTY_SKIP_TESTS[@]:+${TRUSTY_SKIP_TESTS[@]}} \
+          "Monitoring\sshould\sverify\smonitoring\spods\sand\sall\scluster\snodes\sare\savailable\son\sinfluxdb\susing\sheapster" \
```

Comment:
So if we want to have different group, I vote for creating:
TRUSTY_RELEASE_SKIP_TESTS
or sth like that, so we are using the same pattern for everything.